arrow/arrow.mojo

```diff
@@ -19,17 +18,15 @@ struct Bitmap:
         var num_bytes_with_padding = (
             (num_bytes + PADDING - 1) // PADDING
         ) * PADDING
-        var ptr = Pointer[UInt8].alloc(
```

Comment:
First of all thanks for opening the first pull request on the repo!

I don't think we can use UnsafePointer just yet since it doesn't let you specify alignment from `.alloc(..)`. 

`LegacyPointer` on the other hand does have an `alloc` method that lets you specify alignment: https://docs.modular.com/mojo/stdlib/memory/unsafe/LegacyPointer#alloc